test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-post-editor-tracking-spec.js

```diff
@@ -42,6 +44,19 @@ describe( `[${ host }] Calypso Gutenberg Post Editor Tracking: (${ screenSize })
 			postType: 'post',
 		} );
 
+		it( 'Tracks "wpcom_block_editor_details_open" event', async function () {
```

Comment:
Just curious if there is any reason we decided to do this for the 'post' editor tracking spec vs. the 'page' editor.  The both have the details tab while the site editor doesn't.  This seems present in all instances of the post editor regardless of post type, so it may be redundant to check it in multiple places.  Do you think there would be any benefit in adding this to the page tracking spec as well?

Reply:
Actually I didn't even realize we have that in page editor. For some reason I thought this is only available for `post` post_type.

Well, we can track this in the page editor too. It is redundant but might save us in the future.

How about adding this in `general-tests` with a check for `edit-post`? (so we don't run it for `edit-site` but we run it for both post and page).

Reply:
That seems reasonable!  Although, are they still 'general tests' if we have to exclude specific editors from some of the tests? 🤔